CMakeLists.txt

```diff
@@ -691,6 +754,8 @@ if(CATKIN_ENABLE_TESTING)
                           "*/${PROJECT_NAME}/src/psen_scan_driver.cpp"
                           "*/ZoneSet.h" # generated message
                           "*/ZoneSetConfiguration.h" # generated message
+                          "*/PinState.h" # generated message
```

Comment:
this msg doesnt exist anymore. coverage succeeds. how is our policy here? Do we exclude all generated messages per default?

Reply:
Should only succeed with colcon.